### ISSUE TITLE
bootstrap broken in f90aeb0?

### ISSUE THREAD

**Issue by tuncer** [haskell/cabal#4932]
Is bootstrap broken in f90aeb0e53fdd337e7528b1ebbdd39c2d8064637?

Using 8.2.2 on amd64-linux.

```
[...]
Registering library for hackage-security-0.5.2.2..
[1 of 1] Compiling Main             ( Setup.hs, Setup.o )
Linking Setup ...
Configuring cabal-install-2.1.0.0...
Setup: Encountered missing dependencies:
HTTP >=4000.1.5 && <4000.4,
async >=2.0 && <3,
bytestring >=0.9 && <0.10.2,
bytestring-builder >=0.10 && <1,
directory ==1.1.*,
hashable >=1.0 && <2,
network >=2.4 && <2.6,
process >=1.0.1.1 && <1.1.0.2

Error during cabal-install bootstrap:
Configuring the cabal-install package failed.
```

**Comment by hvr** [comment]
Well, our CI seems to test bootstrapping only with GHC 8.0.2... so I wouldn't be surprised if the shell script fell behind for GHC 8.2.2

**Comment by tuncer** [comment]
How about #4933, then?

**Comment by 23Skidoo** [comment]
Fixed by #4933.
